bitcraze_crazyflie_2/README.md

```diff
@@ -0,0 +1,20 @@
+# Bitcraze Crazyflie2 Description (MJCF)
+
+Tested with MuJoCo 3.1.2
+
+## Overview
+
+This package contains  simplified robot description (MJCF) of the Crazyflie2 model from the [Bitcraze](https://www.bitcraze.io/). It is derived from the publicly available, but archived, ROS description: [crazyflie_ros](https://github.com/whoenig/crazyflie_ros).
```

Comment:
Crazyflie2 model from Bitcraze (remove the). Also no need to mention the repo is archived.